docs/Getting Started/Workspaces.md

```diff
@@ -0,0 +1,244 @@
+## What are workspaces?
+
+Polly workspaces allow you to reduce clutter and organize your data in a folder system of choice. This can be accessed by clicking on the *Workspaces* card.
+
+![Homepage](../img/Workspace/1.png) <center>**Figure 1.** Polly Homepage</center>
+
+There are 3 panels on the workspace interface.
+
+*    The left panel allows you to create a new workspace and access all your workspaces.
+
+*    The middle panel shows you all the content within the selected workspace.
+
+*    The panel on the right shows you the details around the content within the workspace.
+
+![Panels](../img/Workspace/2.png) <center>**Figure 2.** Workspace Panels</center>
+
+## Creating a workspace
+
+You can create a new workspace on Polly by clicking on *New Workspace* on the left panel.
+
+![New Workspace1](../img/Workspace/3.png) <center>**Figure 3.** Creating New Workspace</center>
+
+Enter the *Name of the workspace* and *description* (if any) and click on *save*. The new workspace will be created. You can find the newly created workspace with the list of all your workspaces on the left panel.
+
+![New workspace2](../img/Workspace/4.png) <center>**Figure 4.** Workspace Description</center>
+
+The information about the workspace can be viewed through the top panel by clicking on the *Info* icon.
+
+![Workspace Information](../img/Workspace/5.png) <center>**Figure 5.** Workspace Information</center>
+
+
+## Content within a workspace
+
+A workspace contains all your data, analyses, Polly notebooks, and reports. They can be accessed through the middle panel.
+
+![Workspace Content](../img/Workspace/6.png) <center>**Figure 6.** Workspace Contents</center>
+
+
+## Adding files and notebooks to your workspace
+
+*    Using GUI
+
+To add files to your workspaces, click on the *NEW* option present above the middle panel. You can choose the *File Upload* or *Notebook upload* option from within the menu in order to upload the data files and notebooks respectively.
+
+**Note:**
+If your file size is greater than 100 MB, use the Polly CLI option to upload to the workspace.
```

Comment:
just confirm if the point under notes are bullet points or not. also 100mb should be in bold.